### PR TITLE
Implemented getHeight and setHeight on AbstractButtonWidget

### PR DESCRIPTION
**Pull request by bs2kbs2k**
Implemented https://github.com/PatchworkMC/YarnForge/blob/1.14.x/patches/minecraft/net/minecraft/client/gui/widget/AbstractButtonWidget.java.patch#L21
found on issue #125.
Change is very trivial and doesn't need testing.

### CODE REVIEW

File: patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import net.minecraft.client.gui.DrawableHelper;
+import net.minecraft.client.gui.widget.AbstractButtonWidget;
+
+@Mixin(AbstractButtonWidget.class)
+public abstract class MixinAbstractButtonWidget extends DrawableHelper {
+	@Shadow
+	protected int height;
+
+	public int getHeight() {
+		return this.height;
```
Comment (famous1622) [inline patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java:34]: Can these just be Accessors?

**Comment by YTG1234** [comment]
In the future, when Patchwork is complete, I want to be able to use _some_ Forge things in my Fabric mods (aka make 🦆 interfaces please)

**Review by YTG1234 — APPROVED** [review]
Other than that, looks good

File: patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java (hunk 1 above)
Comment (williambl) [reply patchwork-gui/src/main/java/net/patchworkmc/mixin/gui/MixinAbstractButtonWidget.java:34]: Agreed.

**Review by TheGlitch76 — APPROVED** [review]
I'm 95% sure Mixin mangles accessors, so this is good how it is.
Thanks for contributing!

**Pull request merged**